CHANGELOG.md

```diff
@@ -5,21 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [6.0.0-dev21]
+## [6.0.0-dev20]
```

Comment:
I added this dev21 section recently, but we haven't cut dev20 yet so I shouldn't have. This removes the extra increment, and adds a note on this change to dev20.